### DISCUSSION TITLE
「开源之史」系列之二/六：文化的重要作用 —— 开源的本质 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#76] [Announcements]
# 「开源之史」系列之二/六：文化的重要作用 —— 开源的本质 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/04-00-culture-matters-os/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
这次总算有了然后, 
越来越象标准软文了, 有比喻, 有结合, 有小结,
不过, 说服力不足,

毕竟将 卡路里 和开源运动 类比度不足;

可以找个独有的类比点, 进行广泛的使用, 模式也是现成的,
就用外媒的标准结构: 这国怎/总亏民/我陷思/定体问..
具体套用具体案例:
如果让评价某个案件/事件:

- 这个国家怎么了
- 为什么吃亏的总是公民？
- 我不禁陷入了沉思，这一定是体制问题..
然后话题一转就去骂总统和国会山的老爷们就行;-)

现在开源的所有推广论断都太文艺了, 根本难以激发广大人民的情绪...

开源的本质是什么? 不过是程序猿和普通人一样的 

> 追求向往的生活

生活不应该只有苟且和屎山代码, 
我们想创造出令自己自豪的作品,
然后就开始了...

其它一切阻碍, 都可以用开源作品来隔断,
只是, 代码之外的世界, 在持续入侵创造开源世界的人的生活,
这才导致了无力感...

这些, 对开源而言是超纲内容, 感觉当前对开源的宣传, 得同时来:

- 对外: `本正清源` 给开源正名
- 对内: `人情世故` 给开源人扫盲各种现实世界武器知识, 以便可以自由调用的相关社区资源, 共同对抗在不断污染开源世界的资本世界...

**Reply by lijiangsheng1** [reply]
《开源之史》不应该承受这么多的东西～ :-)   发现开源三部曲也不过是作者的观察和思考。 想影响另外的人，不敢奢求。
